méli-mélo/méli-mélo-en.html

```diff
@@ -0,0 +1,95 @@
+
+{% assign melimelo_pages = site.pages | where: "output", "false" | where: "feature", "méli-mélo" | sort: "componentName" %}
+
+<!--
+Number of méli-mélo: {{ melimelo_pages | size }}
+-->
+
+<ul>
+{% for feature in melimelo_pages %}
+	{% assign first_example = feature.pages.examples | first %}
+	<li><strong>{{ feature.componentName }}</strong>: <a href="{{ feature.componentName }}/{{ first_example.path }}" lang="{{ first_example.language }}">{{ first_example.title }}</a></li>
+{% endfor %}
+</ul>
+
+<h2>Requirement</h2>
+
+<ul>
+	<li>Be reusable</li>
+	<li>Be sponsored by a department and by an active representative of that department</li>
+	<li>Must include an implementation plan that explain the steps taken by the sponsored department on how they will actively work the feature to get it integrated into GCWeb.</li>
+	<li>Must no contain any interference with wet-boew and GCWeb.</li>
+	<li>Each individual sub-feature and style must be demoed.</li>
+	<li>Working demo must be published using the GCWeb Jekyll template</li>
+	<li>Project name must be in this format. Year and month of the publication time of the feature followed by a short name identifying the feature</li>
+</ul>
+
+<h2>Contribution instruction</h2>
+
+<ul>
+	<li>Code or/and re-organize the feature by levering GCWeb Jekyll theme</li>
+	<li>Create a project folder in the <code>méli-mélo</code> GCWeb root folder</li>
+	<li>Loading the feature must not change the state of generic page. Feature must be explicitly activated through HTML by the web publisher, like the wet-boew plugin or the usage of a CSS class onto an HTML element.</li>
+	<li>The project name must follow this notation: year-month-projectName. The year and month must correspond to the feature initial publication date. For example "2021-05-steps"</li>
+	<li>The implementation plan should be published along with the project web pages. Each item of the implementation plan must be dated because we are going to use to measure the integration progress into GCWeb. The expectation is to get the méli-mélo feature fully integrated into GCWeb within less than one (1) year.</li>
+	<li>Test your code and submission by using your GitHub pages</li>
+	<li>Submit the méli-mélo feature through a new GitHub PR in the GCWeb project.</li>
+	<li>Once approved by the technical review, it will be assigned to a méli-mélo compilation and released on Canada.ca at the next release windows between 1 to 3 weeks.</li>
+	<li>Update the méli-mélo feature code by executing the implementation plan and addressing each todos identified by the wet-boew technical review team.</li>
+	<li>Participate at the weekly wet-boew Tuesday afternoon code sprint</li>
+	<li>Create the GCWeb provisional/stable feature aligned with our enterprise design approach.</li>
+</ul>
+
+<h2>Technical review checklist</h2>
+
+<ul>
+	<li>Ensure the project sponsor are clearly identified.</li>
+	<li>Project folder name follow the naming convention</li>
```

Comment:
"name follow" => "name follows" (third-person singular conjugation for "folder name").